lib/config-loader.js

```diff
@@ -135,15 +135,18 @@ module.exports = () => {
   // 4. Load manifest config
   if (config.app.hasBackend) {
     config.manifest.packagePlaceholder = '__APP_PACKAGE__'
+    // TODO: migrate existing projects that have the packagePlaceholder in their manifest.yml
     config.manifest.full = yaml.safeLoad(fs.readFileSync(config.manifest.src, 'utf8'))
-    config.manifest.package = config.manifest.full.packages[config.manifest.packagePlaceholder]
+    const packageNames = Object.keys(config.manifest.full.packages)
+    config.manifest.package = config.manifest.full.packages[packageNames[0]]
```

Comment:
another proposition 🙂  (requires more changes, maybe good for a later PR): 
Now that we are going into the direction of full manifest control, I think it doesn't make sense to consider only 1 package anymore, especially now that we don't have the placeholder anymore.
I would suggest getting the action list from all packages. In the long run we could even reuse some parsing logic from the runtime plugin/future runtime lib

Reply:
+1

Reply:
all packages are currently being deployed, they just don't all have an output url currently.

Reply:
true but actions in other packages don't have debug configs, nor will they be listed in the UI template and the filter option won't work neither (`aio app deploy -a deployonlythisaction`).